### DISCUSSION TITLE
SSL/TLS error on script to install RustDesk

### DISCUSSION THREAD

**Discussion by jrbcast** [amidaware/tacticalrmm#1927] [General]
After a long and fast help from dinger1986, I managed to make Rustdesk work from tactical RMM (love it). Now, I was desploying the client to my already added servers and most of them are doing right but two (at the moment). It is complaining with this error: 

```
Invoke-WebRequest : Anulada la solicitud: No se puede crear un canal seguro 
SSL/TLS.
En lnea: 1 Carcter: 1
+ Invoke-WebRequest 
https://github.com/rustdesk/rustdesk/releases/download/1.2.2/r ...
+ CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:Htt 
   pWebRequest) [Invoke-WebRequest], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShe 
   ll.Commands.InvokeWebRequestCommand
 
```

Script timed out after 93 seconds

Both are Windows Server 2012 and 2012 R2. Other servers running Windows Server 2016 and Windows Server 2019 have done right so far... 

I have found some guiding posts: 

https://stackoverflow.com/questions/36265534/invoke-webrequest-ssl-fails/36266735#36266735

But have not had look making it work. Also, I would not like to modify scripts myself in case the next Tactical release those chance. 

Any hint?

Thanks in advance, 
Jose

**Comment by silversword411** [comment]
server 2012 doesn't have TLS 1.2, add it to your server support

**Reply by jrbcast** [reply]
As easy as that... thanks!
